lib/DDG/Spice/Piratetalk.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Piratetalk;
+
+# ABSTRACT: Piratetalk will translate texts into Pirate language when triggered with the trigger words mentioned below. 
+# For example, "Pirate Speak Hello" will return "Ahoy!" 
+
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; 
+spice wrap_jsonp_callback => 1; 
+
+spice to => 'https://pirate-speak.herokuapp.com/piratespeak?text=$1';
+
+triggers any => 'pirate talk', 'pirate speak', 'translate to pirate';
```

Comment:
We're going to need to reign in these triggers so they better indicate intent to translate.

The top auto-complete results for "pirate talk" and "pirate speak" aren't attempting to translate a word/phrase.

triggers like "X to pirate speak", "pirate speak for X", "translate to pirate talk", etc are better.

you can use both `triggers start => '...'` and `triggers end => '...'` and even `triggers startend => '...'` to define several different triggers if you'd like